src/features/world/ui/NPCModals.tsx

```diff
@@ -184,6 +131,26 @@ export const NPCModals: React.FC<Props> = ({ onClose, onOpen }) => {
             </div>
           </CloseButtonPanel>
         )}
+        {/* Delivery NPC's */}
+        {npc === "pumpkin' pete" && (
+          <DeliveryModal npc={npc} onClose={closeModal} />
+        )}
+        {npc === "blacksmith" && (
```

Comment:
After completing the delivery there was no `tap to continue` to close the modal.

I could click outside, but I feel keeping `tap to continue` is nice since players will be tapping on these modals to advance